package.json

```diff
@@ -13,8 +13,10 @@
     "release:dry-run": "semantic-release --dry-run",
     "start": "NODE_ENV=development nodemon scripts/start.js --exec babel-node",
     "test": "jest",
-    "test:integration": "jest \"^(?=.*integration.test.js).*\"",
+    "test:integration": "jest \"^(?=.*RoomsSDKAdapter.integration.test.js).*\"",
+    "test:integration:coverage": "npm run test:integration --coverage",
```

Comment:
Can't you just add the flag on the CI instead of creating another script for each?

Reply:
yeah I will do that.